### ISSUE TITLE
Sort order of Track feature

### ISSUE THREAD

**Issue by diesl** [OpherV/gitflow4idea#290]
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request

* **What is the current behavior?**
When I open *Track Feature*, I see all available feature branches but with no obvious ordering.

* **What is the expected behavior?**
I would like to have the branches ordered by name, like in the *Git Branches* window. Another solution is to let the user select the desired sort order

* **What is the motivation / use case for changing the behavior?**
Faster selection of branch. Currently I have to scan the whole list to find the correct feature branch

* **Please tell me about your environment:**
  
  - Gitflow4idea version: 0.7.1
  - Gitflow version: 1.11.0 (AVH Edition)
  - RubyMine
  ~~~~
    RubyMine 2019.3.4
    Build #RM-193.6911.24, built on March 17, 2020
    Runtime version: 11.0.6+8-b520.43 amd64
    VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
  ~~~~

**Comment by OpherV** [comment]
Good idea. Would also be nice to have search functionality.

**Comment by OpherV** [comment]
Fix in 0.7.3 now out

**Comment by diesl** [comment]
Works perfect! Thanks 👍
